frameworks/cassandra/universe/resource.json

```diff
@@ -7,7 +7,7 @@
       "cassandra-openssl-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/openssl-1.0.2n-libs.tar.gz",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
-      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.16-bin-dcos.tar.gz",
+      "cassandra-tar-gz": "https://s3-ap-southeast-2.amazonaws.com/cassandra-lcm-bucket/apache-cassandra-3.0.17.tar.gz",
```

Comment:
This artefact *must* be available in the `downloads.mesosphere.com` before this can be merged.

Reply:
We are getting access denied error hence unable to upload to downloads.mesosphere.com. We are following up with mesosphere team and will keep you posted.

Reply:
please review : @realmbgl @elezar 
cc : @ryadav88